test/system/505-networking-pasta.bats

```diff
@@ -766,8 +768,21 @@ function pasta_test_do() {
     local pidfile="${PODMAN_TMPDIR}/pasta.pid"
 
     run_podman run --rm "--net=pasta:--pid,${pidfile}" $IMAGE true
-    sleep 1
-    ! ps -p $(cat "${pidfile}") && rm "${pidfile}"
```

Comment:
We had a flake. It was very painful to debug. I've rewritten it to be more robust and also to make things easier for the next maintainer.